plugins/modules/device_credential_workflow_manager.py

```diff
@@ -1961,7 +1966,7 @@ def get_want_assign_credentials(self, AssignCredentials):
         site_id = []
```

Comment:
We can rename to site_ids to avoid conflict and indicate it's a collection ?
site_ids = []